conf/distro/include/raspberrypi_generic.conf

```diff
@@ -12,10 +12,11 @@ PACKAGECONFIG_append_pn-wpeframework-plugins = "\
 
 PACKAGECONFIG_append_pn-wpeframework-plugins-rdk = " messenger packager"
 
-PREFERRED_VERSION_wpewebkit = "2.22%"
-PREFERRED_VERSION_libwpe = "1.0.0%"
-PREFERRED_VERSION_wpebackend-rdk = "git%"
+IMAGE_FSTYPES = "tar.xz ext3 rpi-sdimg"
```

Comment:
This should be moved to the recipes-core/images/wpe-image.inc

Reply:
done